src/collaboration-platform/CollaborationPlatform.ts

```diff
@@ -31,7 +31,11 @@ export interface CollaborationPlatform {
    * @see updatePRDescriptionsForCommitGraph
    */
   createOrUpdatePRForCommits(
-    commitBranchPairs: { commit: Commit; branch: BranchName }[],
+    commitBranchPairs: {
+      commit: Commit;
+      headBranch: BranchName;
+      baseBranch: BranchName; //NOTE: add the base branch
```

Comment:
Not really "pairs" anymore 😆

@manyaagarwal: this method now needs a based branch as discussed on Discord

```suggestion
    commitsWithMetadata: {
      commit: Commit;
      headBranch: BranchName;
      baseBranch: BranchName;
```